### ISSUE TITLE
Metadata-based value imputation

### ISSUE THREAD

**Issue by rgerkin** [neuroelectro/neuroelectro_org#333]
Have a model on the server which can impute the data values that would be expected under a provided set of metadata (or a standard set of metadata), and return those.  

@dtebaykin Have you worked on a model that might satisfy these properties, or is this something I should implement (I would implement the API part of this either way).

**Comment by dtebaykin** [comment]
Yes, I am working on exactly that. We will need to figure out how to get that on Neuroelectro though.
